.enb/lib/page.js

```diff
@@ -0,0 +1,149 @@
+var techs = require('./techs');
+
+module.exports = function(node, options) {
+    var levels = options.levels || [],
+        browsers = options.browser || [],
+        langs = options.langs || [],
+        templateEngine = options.templateEngine || 'BH';
+
+    // essential
+    node.addTechs([
+        [techs.bem.levels, { levels : levels }],
+        [techs.bem.bemjsonToBemdecl, {
+            target : '.tmp.bemdecl.js'
+        }],
+        [techs.bem.deps, {
+            bemdeclFile : '.tmp.bemdecl.js',
+            target : '.tmp.deps.js'
+        }],
+        [techs.bem.files, {
+            depsFile : '.tmp.deps.js'
+        }]
+    ]);
+
+    // css
+    node.addTech([techs.css, {
+        target : '.tmp.css',
+        browsers : browsers
+    }]);
+    node.mode('development', function() {
+        node.addTech([techs.files.copy, {
+            source : '.tmp.css',
+            target : '?.css'
```

Comment:
I reopened old [issue](https://github.com/enb-bem/enb-bem-techs/issues/76) in this topic. I propose to continue the discussion there.